CITATION.cff

```diff
@@ -93,6 +93,10 @@ authors:
   affiliation: "University of Gent"
   orcid: "https://orcid.org/0000-0001-8279-6118"
 
+- family-names: Huber
+  given-names:  Daniel
+  affiliation: University of Innsbruck
```

Comment:
@d-ni374 
is this your affiliation? (got this from github?)

Reply:
I will merge the PR, but do not hesitate to open another PR to amend this

Reply:
Yes, the affiliation is correct. Thank you for adding!